src/pages/pt-BR/reference/configuration-reference.md

```diff
@@ -0,0 +1,371 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Referência da Configuração
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+
+A referência a seguir cobre todas as opções de configuração suportadas no Astro. Para aprender mais sobre como configurar o Astro, leia o nosso guia, [Configurando Astro](/pt-BR/guides/configuring-astro/).
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config'
+
+export default defineConfig({
+  // suas opções de configuração aqui...
+})
+```
+## Opções Top-Level
+
+### root
+
+<p>
+
+**Tipo:** `string`<br>
+**Interface de Linha de Comando:** `--root`<br>
+**Padrão:** `"."` (diretório de trabalho atual)
+</p>
+
+Você deve apenas providenciar esta opção se você executar os comandos da interface de linha de comando `astro` em um diretório diferente do que o diretório raiz do projeto. Geralmente, esta opção é providenciada pela interface de linha de comando ao invés do arquivo `astro.config.js`, já que Astro precisa saber a raiz do seu projeto antes de poder localizar seu arquivo de configuração.
+
+Se você providenciar um caminho relativo (ex: `--root: './meu-projeto'`) Astro irá resolvê-lo com base no seu diretório de trabalho atual.
+
+#### Exemplos
+
+```js
+{
+  root: './diretorio-do-meu-projeto'
+}
+```
+```bash
+$ astro build --root ./diretorio-do-meu-projeto
+```
+
+
+### srcDir
+
+<p>
+
+**Tipo:** `string`<br>
+**Padrão:** `"./src"`
+</p>
+
+Define o diretório em que Astro irá ler o seu site. 
+
+O valor pode ser tanto um caminho absoluto do sistema ou um caminho relativo a raiz do projeto.
+
+```js
+{
+  srcDir: './www'
+}
+```
+
+
+### publicDir
+
+<p>
+
+**Tipo:** `string`<br>
+**Padrão:** `"./public"`
+</p>
+
+Define o diretório de seus assets estáticos. Arquivos nesse diretório são servidos em `/` durante o desenvolvimento e são copiados para o diretório da sua build durante a build. Esses arquivos serão sempre servidos ou copiados da forma que são, sem transformações ou etapa de bundle.
```

Comment:
It changes a little bit, but these `build` repetition is a little bit confusing in portuguese.

```suggestion
Define o diretório de seus assets estáticos. Arquivos nesse diretório são servidos em `/` durante o desenvolvimento e são copiados para o diretório de saída durante o processo de build. Esses arquivos serão sempre servidos ou copiados da forma que são, sem transformações ou etapa de bundle.
```